File: Practicas/0. Ejemplos Code/Ejemplo.c

int suma(int a, int b) {
    return a + b;
}

int main() {
    int a = 2;
    char* b = "3";

    printf("Voy a sumar %d y %s\n", a, b);
    printf("¿debería ser 5?\n");

    printf("%d\n", suma(a, b));

    return 0;
}

File: Practicas/0. Ejemplos Code/Ejemplo.java
public class Ejemplo {
  public static int suma(int a, int b) {
    return a + b;
  }

  public static void main(String[] args) {
    int a = 2;
    int b = 3;

    System.out.printf("Voy a sumar %d y %s%n", a, b);
    System.out.println("deberia ser 5?");
    System.out.println(suma(a, b));
  }
}